class-cr-clone.php

```diff
@@ -64,7 +64,18 @@ public function __action_admin_post() {
 			wp_die( esc_html__( 'There was an error copying this post', 'clone-replace' ) );
 		}
 
-		wp_redirect( admin_url( "post.php?post={$post_id}&action=edit" ) );
+		/**
+		 * Filter Cloned Post Redirect URL.
+		 *
+		 * Modify the resulting redirect location after cloning a post.
+		 *
+		 * @since 0.2
+		 *
+		 * @param string $redirect_url URL string passed to wp_redirect
+		 */
+		$redirect_url = apply_filters( 'CR_Clone_redirect_url', admin_url( "post.php?post={$post_id}&action=edit" ) );
```

Comment:
Two more little things:
* [ ] Pass the post id, so other code hooking into this doesn't have to parse it from the url (don't forget to document the `@param` too)
* [ ] This line is too long, so break it up, e.g.
   ```php
   apply_filters(
      'CR_Clone_redirect_url',
      admin_url( ... ),
      $post_id
   );
   ```